dts/bindings/serial/xen,hvc-uart.yaml

```diff
@@ -0,0 +1,9 @@
+description: Xen Platform HVC UART
+
+compatible: "xen,hvc-uart"
+
+include: uart-controller.yaml
+
+properties:
+    reg:
+      required: false
```

Comment:
If this is `false` there is no need to have the `properties` section at all, since the default for `reg` property will be `required: false`

Reply:
sure, done it